api/Request.json

```diff
@@ -1273,47 +1273,47 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Request/referrerPolicy",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "51"
```

Comment:
For Request, this actually landed in 52.

https://chromium.googlesource.com/chromium/src/+/a8e98a61a6144db6a2b45baaff9eeb8930f0296f
https://storage.googleapis.com/chromium-find-releases-static/a8e.html#a8e98a61a6144db6a2b45baaff9eeb8930f0296f

Reply:
OK, updated.